infra/gravity/cluster_resize.go

```diff
@@ -68,22 +68,24 @@ func (c *TestContext) ShrinkLeave(nodesToKeep, nodesToRemove []Gravity) error {
 	return trace.Wrap(utils.CollectErrors(ctx, errs))
 }
 
-// RemoveNode simulates sudden nodes loss within an existing cluster followed by node eviction
-func (c *TestContext) RemoveNode(nodesToKeep []Gravity, remove Gravity) error {
+// RemoveNodes evicts several nodes from the cluster
```

Comment:
The previous comment mentioned a "sudden node loss" but it seems like it's just doing regular "gravity remove"?

Reply:
The previous version of this wasn't doing a sudden node loss either.  The way that `loss_recover` calls this function conditionally would power off the node before hand.  Either way, the previous comment was misleading, which is why I updated the comment.